docs/users/cli-usage.md

```diff
@@ -0,0 +1,146 @@
+<h1>Using the ddev command line interface (CLI)</h1>
+
+Type `ddev` in a terminal to see the available ddev commands:
+
+```
+➜  ddev
+This Command Line Interface (CLI) gives you the ability to interact with the ddev to create a local development environment.
+
+Usage:
+  ddev [command]
+
+Available Commands:
+  config       Create or modify a ddev application config in the current directory
+  describe     Get a detailed description of a running ddev site.
+  exec         Execute a Linux shell command in the webserver container.
+  hostname     Manage your hostfile entries.
+  import-db    Import the database of an existing site to the local dev environment.
+  import-files Import the uploaded files directory of an existing site to the default public upload directory of your application.
+  list         List applications that exist locally
+  logs         Get the logs from your running services.
+  restart      Restart the local development environment for a site.
+  remove       Remove an application's local services.
+  sequelpro    Easily connect local site to sequelpro
+  ssh          SSH to an app container.
+  start        Start the local development environment for a site.
+  stop         Stop an application's local services.
+  version      print ddev version and component versions
+
+Use "ddev [command] --help" for more information about a command.
+```
+
+
+## Getting Started
+Check out the git repository for the site you want to work on. `cd` into the directory and run `ddev config` and follow the prompts.
+
+```
+$ cd ~/Projects
```

Comment:
I'm not opposed to using a demonstration project that's not ours, but the question then becomes what to use. I don't think using the drupal or wp git mirrors is a good route, as you will either have to demonstrate a clone for a stable tag/branch, or risk the user trying to spin a broken master. We could alternatively make this a non-working example and change to `git clone [your project]`